bin/content-style

```diff
@@ -1,30 +1,32 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-$LOAD_PATH.unshift File.join(File.dirname(__FILE__), *%w(.. lib))
-
 require 'content_style'
-require 'YAML'
-
-command = ARGV[0]
+require 'yaml'
```

Comment:
Gems are typically lower + snake case in name (but for some reason uppercase works locally?)